services/system/AuthSig/src/RAuthSig.cpp

```diff
@@ -31,6 +31,8 @@ namespace SystemService
                          optional<string>        before,
                          optional<string>        after) const
          {
+            std::erase(pubkeyPem, '\n');
```

Comment:
I'm actually somewhat surprised this works. The new line (CR, LF, or CRLF) is [required](https://www.rfc-editor.org/rfc/rfc7468#page-5) for PEM, though many parsers are somewhat forgiving. The reason it failed, was that the query was not correct.

Reply:
Changed my approach